nix/buildRocPackage.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs, roc-cli, name, entryPoint, src, outputHash, linker ? "", ... }:
```

Comment:
Right now this depends on passed in `roc-cli`. I think it's fine until roc is in nixpkgs, then we could just use `roc-cli` from `pkgs`.